swift-sdk/Internal/OnlineRequestProcessor.swift

```diff
@@ -253,6 +253,16 @@ struct OnlineRequestProcessor: RequestProcessorProtocol {
                     requestIdentifier: "trackEmbeddedMessageClick")
     }
     
+    @discardableResult
+    func track(embeddedMessageDismiss message: IterableEmbeddedMessage,
+               onSuccess: OnSuccessHandler? = nil,
+               onFailure: OnFailureHandler? = nil) -> Pending<SendRequestValue, SendRequestError> {
+        sendRequest(requestProvider: { apiClient.track(embeddedMessageDismiss: message) },
+                    successHandler: onSuccess,
+                    failureHandler: onFailure,
+                    requestIdentifier: "trackEmbeddedMessageClick")
```

Comment:
Should be "trackEmbeddedMessageDismiss".

Reply:
Fixed